app/src/pages/inside/projectSettingsPageContainer/content/notifications/modals/deleteNotificationCaseModal/deleteNotificationCaseModal.jsx

```diff
@@ -53,16 +53,19 @@ const DeleteNotificationCaseModal = ({ data: { onSave } }) => {
         text: formatMessage(COMMON_LOCALE_KEYS.CANCEL),
       }}
       onClose={() => dispatch(hideModalAction())}
+      isFieldsChanged={dirty}
```

Comment:
When you rename the property to `allowCloseOutside` and set it to `true` by default, the value here should be `allowCloseOutside={!dirty}` so that the user can close the modal outside of it if no changes have been made. .